policy/modules/admin/bootloader.te

```diff
@@ -41,10 +41,11 @@ dev_node(bootloader_tmp_t)
 # bootloader local policy
 #
 
-allow bootloader_t self:capability { chown dac_override dac_read_search fsetid mknod setgid sys_admin sys_rawio };
+allow bootloader_t self:capability { chown dac_override dac_read_search fsetid sys_chroot mknod setgid sys_admin sys_rawio };
 dontaudit bootloader_t self:capability sys_resource;
-allow bootloader_t self:process { signal_perms execmem };
+allow bootloader_t self:process { signal_perms execmem getsched setfscreate };
 allow bootloader_t self:fifo_file rw_fifo_file_perms;
+allow bootloader_t self:netlink_selinux_socket connected_socket_perms;
```

Comment:
I need more information on the bootloader changes. There are too many and there is no explanation in this PR. I didn't get any results of `grep -i selinux` in the grub source code. The only result was in `tests/util/grub-fs-tester.in` for mount options.

Reply:
I'll back that out and resubmit when I find a good explanation